pristas-peter/gatsby-wordpress-gutenberg
# Feature/16 core theme

See: #17 

I have closed the previous PR in favour of this, because I could not request a review, because it was not opened by myself.

How to test this PR:

- remove the db folder containing database and run docker-compose again
- database should be seeded with testing data
- explore the sources of gatsby-theme-wordrpress-gutenberg - especially the gatsby-node.js
- read the comments in that file
- explore the sources of example project - especially the src/gatsby-theme-wordpress-gutenberg
- spin up the frontend - note the imports of the default wordpress block stylesheet
- try to modify files inside src/gatsby-theme-wordpress-gutenberg - the plugin should handle add/edit/delete of files in components and templates folders
- try to create new block component (check out the quote example)
- try to create new template component (check out the usage in example project)
- try to edit blocks in gutenberg admin, publish changes and run curl -X POST http://localhost:8000/__refresh while the gatsby develop is still running - plugin should handle all the updates without errors
- leave questions :)
